app/bundles/LeadBundle/Tests/Deduplicate/ContactMergerFunctionalTest.php

```diff
@@ -14,11 +14,11 @@ final class ContactMergerFunctionalTest extends MauticMysqlTestCase
 {
     public function testMergedContactFound(): void
     {
-        $model = self::$container->get('mautic.lead.model.lead');
-        \assert($model instanceof LeadModel);
+        /** @var LeadModel $model */
+        $model = self::getContainer()->get('mautic.lead.model.lead');
```

Comment:
The assert() function is preferred over a comment typehint as it will actually fail on test and dev environments yet has no affect on prod environments.

Other occurrences in this file.